src/site/content/en/learn/css/selectors/index.md

```diff
@@ -0,0 +1,486 @@
+</figure>
+
+This effect is better visualised in the above example,
+using the combinator selector, `.top div`.
+That CSS rule adds left padding to those `<div>` elements.
+Because the combinator is recursive,
+all `<div>` elements that are in `.top` will have that same padding applied to them.
+
+Take a look at the HTML panel in this demo
+to see how the `.top` element has several `<div>` child elements which themselves,
+have `<div>` child elements.
+
+#### Next sibling combinator
+
+You can look for an element that immediately follows another element
+by using a `+` character in your selector.
+
+{% Codepen {
+    user: 'web-dot-dev',
+    id: 'JjEPzwB'
+} %}
+
+To add space between stacked elements,
+use the next sibling combinator to add space
+*only* if an element is a **next sibling** of a child element of `.top`.
+
+You could add margin to all child elements of `.top`,
+using the following selector:
+
+```css
+.top * {
+	margin-top: 1em;
+}
+```
+
+The problem with this is that because you're selecting every child element of `.top`,
+this rule potentially creates extra, unnecessary space.
+The **next sibling combinator**,
+mixed with a **universal selector** enables you to not only control what elements get space,
+but also apply space to **any element**.
+This provides you with some long-term flexibility,
+regardless of what HTML elements appear in `.top`.
+
+#### Subsequent- sibling combinator
+
+A subsequent combinator is very similar to a next sibling selector.
+Instead of a `+` character however,
+use a `~` character.
+How this differs is that an element just has to follow another element with the same parent,
+rather than being the next element with the same parent.
+
+<figure class="w-figure">
+  {% Codepen {
+    user: 'web-dot-dev',
+    id: 'ZELzPPX'
+  } %}
+  <figcaption class="w-figcaption">Use a subsequent selector along with a `:checked` pseudo class to create a pure CSS switch element.</figcaption>
+</figure>
+
+This subsequent combinator provides a little less rigidity,
+which is useful in contexts like the above sample,
+where we change the colour of a custom switch when its associated checkbox has the `:checked` state.
+
+#### Child combinator
+
+A child combinator (also known as direct descendant)
+allows you more control over the recursion that comes with combinator selectors.
+By using the `>` character, you limit the combinator selector to apply **only** to direct children.
+
+Consider the previous, next sibling selector example.
+The space is added to each **next sibling**,
+but if one of those elements also has **next sibling elements** as children,
+it can result in undesirable, extra spacing.
+
+{% Codepen {
+    user: 'web-dot-dev',
+    id: 'ExZYMJL'
+} %}
+
+To alleviate this problem,
+change the **next sibling selector** to incorporate a child combinator: `> * + *`.
+The rule will now **only** apply to direct children of `.top`.
+
+{% Codepen {
+    user: 'web-dot-dev',
```

Comment:
Unpredictable spacing is mentioned, but this looks the same as the previous example (same effect), perhaps a different demo would be better for the previous one? Or show how the spacing is unpredictable?

Also, I'd encourage a bit more of an obvious styling change in this demo too if possible :)

Reply:
It's much more obvious with the highlighting treatment now 👍